.github/CODEOWNERS

```diff
@@ -0,0 +1,64 @@
+# This list auto requests reviews from the specified org members
+# when a PR that modifies the file in question is opened
+# This list is alphabetized by User -> Filename KEEP IT THAT WAY
+
+# ChangelingRain
+
+/code/__DEFINES/clockcult.dm @ChangelingRain
+/code/datums/antagonists/datum_clockcult.dm @ChangelingRain
+/code/game/gamemodes/blob/* @ChangelingRain
+/code/game/gamemodes/clock_cult/* @ChangelingRain
+/code/game/gamemodes/miniantags/revenant* @ChangelingRain
+/code/game/objects/effects/temporary_visuals/* @ChangelingRain
+/code/modules/reagents/chemistry/reagents/blob_reagents.dm @ChangelingRain
+
+# Cyberboss
+
+/code/__DATASTRUCTURES/globals.dm @Cyberboss
+/code/__DEFINES/components.dm @Cyberboss
+/code/__DEFINES/server_tools.config.dm @Cyberboss
+/code/__DEFINES/server_tools.dm @Cyberboss
+/code/controllers/subsystem/atoms.dm @Cyberboss
+/code/controllers/subsystem/mapping.dm @Cyberboss
+/code/controllers/globals.dm @Cyberboss
+/code/datums/helper_datums/getrev.dm @Cyberboss
+/code/datums/components/* @Cyberboss
+/code/datums/map_config.dm @Cyberboss
+/code/datums/forced_movement.dm @Cyberboss
+/code/datums/holocall.dm @Cyberboss
+/code/modules/admin/verbs/adminhelp.dm @Cyberboss
+/code/modules/admin/verbs/adminpm.dm @Cyberboss
+/code/modules/server_tools/* @Cyberboss
+/code/modules/mapping/* @Cyberboss
+
+# duncathan
+
+/code/__DEFINES/atmospherics.dm @duncathan
+/code/modules/atmospherics/* @duncathan
+
+# Jordie0608
+
+/SQL/* @Jordie0608
+/code/controllers/subsystem/dbcore.dm @Jordie0608
+/tools/SQLAlertEmail/* @Jordie0608
+
+# MrStonedOne
+
+/code/__DEFINES/MC.dm @MrStonedOne
+/code/__DEFINES/tick.dm @MrStonedOne
+/code/controllers/admin.dm @MrStonedOne
+/code/controllers/master.dm @MrStonedOne
+/code/controllers/failsafe.dm @MrStonedOne
+/code/controllers/subsystem.dm @MrStonedOne
+/code/controllers/subsystem/air.dm @MrStonedOne
+/code/controllers/subsystem/timer.dm @MrStonedOne
+/code/modules/atmospherics/* @MrStonedOne
```

Comment:
mso won't need pinging for the atmos folder now that I am back. he will need it for the air ticker, of course

Reply:
If I could also be pinged for the air ticker, that would be great, since that can be mso's domain or my domain depending on what gets touched.

Reply:
@MrStonedOne of course if you'd like to be pinged that's probably fine but I assume you were just acting in my place when the original pr for this was made

Reply:
ya the `atmos/*` might be have too little SNR for my taste

I do certain optimizations but i'm not quite authoritative on rather or not something is proper.